api/db/migrations/20240229003203_create_class/migration.sql

```diff
@@ -0,0 +1,33 @@
+-- CreateTable
+CREATE TABLE "Class" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "code" TEXT,
+    "professorId" TEXT NOT NULL,
+
+    CONSTRAINT "Class_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "_classStudents" (
+    "A" INTEGER NOT NULL,
+    "B" TEXT NOT NULL
```

Comment:
I like but, what is "A" and "B"? Document these please, it might be easier for someone to know what's next.

Reply:
As commented earlier, this is automatically generated by `Prisma`, I'll go through some documentation to try solve it.